lib/aur-build

```diff
@@ -26,6 +26,11 @@ db_replaces() {
     }'
 }
 
+run_msg() {
+    printf 'Running %s\n' >&2 "$*"
```

Comment:
This is a bit unexpected order of arguments for my taste, personally I would prefer to put `>&2` either right after `printf`, or before everything, or after everything, but not between arguments to `printf` 🙂